config/application.rb

```diff
@@ -64,7 +64,7 @@ class Application < Rails::Application
       config.default = RGeo::Geographic.projected_factory(
         srid:                    4326,
         projection_srid:         4326,
-        projection_proj4:        '+proj=longlat +ellps=WGS84 +datum=WGS84 +no_defs',
```

Comment:
@LocoDelAssembly Only this makes me hesistate on this pull.  What's the difference b/w this and the proposed?

Reply:
That one is not recognized by the new gem version, but 'EPSG:4326' is supposed to be the same (or very close): https://spatialreference.org/ref/epsg/wgs-84/ (click on Proj4 link)

Reply:
Extra info: Original code causes multiple 'proj_crs_get_geodetic_crs: Object is not a CRS' at boot and also, the factory is not actually instantiated so then lots and lots of undefined method for nil:NilClass.